scripts/common.lib.ps1

```diff
@@ -105,12 +105,18 @@ function Install-DotNetCli
     $dotnetInstallPath = Join-Path $env:TP_TOOLS_DIR "dotnet"
     New-Item -ItemType directory -Path $dotnetInstallPath -Force | Out-Null
 
+    $x64Architecture = "x64"
+    if ($env:PROCESSOR_ARCHITECTURE -eq "ARM64")
+    {
+        $x64Architecture = "arm64"
```

Comment:
What about x86, shall we have some similar logic?

Reply:
I don't know, I don't have a device to test it.